backend/app/main.py

```diff
@@ -100,6 +99,11 @@ def update_stats():
     stats.update(apps)
 
 
+@app.post("/update/exceptions")
```

Comment:
This seems unrelated?

Reply:
Yeah, some leftover from my local repo. I will split it to separate PR in the evening.